### DISCUSSION TITLE
20240803

### DISCUSSION THREAD

**Discussion by Mailaender** [OpenHV/OpenHV#1106] [Announcements]
## Features
* Added tracers to instant hit bullets. by @dnqbob in https://github.com/OpenHV/OpenHV/pull/957
* Added new projectile `InstantExplode` (unused). by @GraionDilach https://github.com/OpenHV/OpenHV/pull/993
* Added play map button. by @Mailaender 
* Added a water refill :potable_water: mechanic to the Fire Watch mission. by @Mailaender 
* Added :green_circle: friend vs. :red_circle: foe colors. by @PunkPun https://github.com/OpenHV/OpenHV/issues/935
* Added automatic ordering units to clear resource spots for miner deployment. https://github.com/OpenHV/OpenHV/issues/1075 by @Mailaender 

## Content
* **New Missions:**
  * Fillum Resources Claim by @OcelotWalrus in https://github.com/OpenHV/OpenHV/pull/967
  * Fillum Crater by @OcelotWalrus in https://github.com/OpenHV/OpenHV/pull/968
  * Harbor Defense by @OcelotWalrus in https://github.com/OpenHV/OpenHV/pull/969
  * Asteroid Invasion by @OcelotWalrus in https://github.com/OpenHV/OpenHV/pull/970
  * Scientist :scientist: Rescue by @OcelotWalrus in https://github.com/OpenHV/OpenHV/pull/1036

* **New Conquest Maps:**
  * Woody :wood: Passage (3v3) by @OcelotWalrus in https://github.com/OpenHV/OpenHV/pull/971
  * Swamp Conquest by @OcelotWalrus in https://github.com/OpenHV/OpenHV/pull/995
  * Swampy Frontier map (2v2) by @OcelotWalrus in https://github.com/OpenHV/OpenHV/pull/1025
  * Tyrian Dawn map (3v3) by @OcelotWalrus in https://github.com/OpenHV/OpenHV/pull/1026
  * Dark Waters by @OcelotWalrus in https://github.com/OpenHV/OpenHV/pull/1010
  * Fight At The River by @OcelotWalrus in https://github.com/OpenHV/OpenHV/pull/1034 https://github.com/OpenHV/OpenHV/pull/1054
  * Duel Of The Shades by @OcelotWalrus in https://github.com/OpenHV/OpenHV/pull/1039
  * Lost Eisley :desert: (4P) by @OcelotWalrus in https://github.com/OpenHV/OpenHV/pull/1020 https://github.com/OpenHV/OpenHV/pull/1029
  * A Nuclear Winter (2v2) by @OcelotWalrus in https://github.com/OpenHV/OpenHV/pull/1102
  * Fractional Freezing (1v1) by @SiegeSpud in https://github.com/OpenHV/OpenHV/pull/1080
  * Bay Under The Rain (4P) by @OcelotWalrus

* Added artwork for Lightning Boat. by @Dzierzan in https://github.com/OpenHV/OpenHV/pull/984
* Map update: Clovergried by @SiegeSpud in https://github.com/OpenHV/OpenHV/pull/990

* New behemoth aircraft (not yet enabled)
  * Added Mothership unit. by @Dzierzan in https://github.com/OpenHV/OpenHV/pull/985
  * Added Battleship unit. by @Dzierzan in https://github.com/OpenHV/OpenHV/pull/1000

* Added extra road tiles for diagonal bridges. by @Dzierzan in https://github.com/OpenHV/OpenHV/pull/982
* Added sounds and effects to Televator. by @Dzierzan in https://github.com/OpenHV/OpenHV/pull/1002 and https://github.com/OpenHV/OpenHV/pull/1008
* Update Howitzer's power icon by @OcelotWalrus in https://github.com/OpenHV/OpenHV/pull/1014
* Updated Televator's support power icon. by @Dzierzan in https://github.com/OpenHV/OpenHV/pull/1011
* Updated artwork of Shield's icon. by @Dzierzan in https://github.com/OpenHV/OpenHV/pull/1012
* Updated Banshee's weapon artwork. by @Dzierzan in https://github.com/OpenHV/OpenHV/pull/1016
* Upgraded map Mount Brosnan by @OcelotWalrus in https://github.com/OpenHV/OpenHV/pull/1021
* Added surveillance effect. by @Dzierzan in https://github.com/OpenHV/OpenHV/pull/1057
* Added Torpedo Boat unit. by @Dzierzan in https://github.com/OpenHV/OpenHV/pull/1059
* Enlarged Lightning Tank sprites. by @Dzierzan in https://github.com/OpenHV/OpenHV/pull/1064
* Update builder artwork differing by faction. by @Dzierzan in https://github.com/OpenHV/OpenHV/pull/1072 https://github.com/OpenHV/OpenHV/pull/1074

* **Voiceovers** :studio_microphone: 
  * Re-recorded all announcer voices. by @Dzierzan in https://github.com/OpenHV/OpenHV/pull/1022
  * Re-recorded Stealth Tank voices. by @Dzierzan in https://github.com/OpenHV/OpenHV/pull/1030
  * Re-recorded generic unit voices. by @Dzierzan in https://github.com/OpenHV/OpenHV/pull/1028
  * Increased the quality of flamewh33l's voice lines. by @Dzierzan in https://github.com/OpenHV/OpenHV/pull/1031
  * Increased the quality of generic ship's voice lines. by @Dzierzan in https://github.com/OpenHV/OpenHV/pull/1032
  * Re-recorded Miner voices. by @Dzierzan in https://github.com/OpenHV/OpenHV/pull/1043
  * Re-recorded seed critter voice. by @Dzierzan in https://github.com/OpenHV/OpenHV/pull/1044
  * Re-recorded beast critter voice. by @Dzierzan in https://github.com/OpenHV/OpenHV/pull/1045
  * Re-recorded technician voice. by @Dzierzan in https://github.com/OpenHV/OpenHV/pull/1052
  * Re-recorded Flame Pod voices. by @Dzierzan in https://github.com/OpenHV/OpenHV/pull/1062
  * Added re-recorded generic voice. by @GraionDilach and @Dzierzan in https://github.com/OpenHV/OpenHV/pull/1068
  * Re-recorded Builder voices. by @Dzierzan in https://github.com/OpenHV/OpenHV/pull/1085
  * Added new assault tank voice. by @Dzierzan in https://github.com/OpenHV/OpenHV/pull/1086
  * Re-recorded Electric Pod voices. by @Dzierzan in https://github.com/OpenHV/OpenHV/pull/1092
  * Re-recorded Civilian Pod voices. by @Dzierzan in https://github.com/OpenHV/OpenHV/pull/1095
  * Re-recorded Elite Pod voices. by @Dzierzan in https://github.com/OpenHV/OpenHV/pull/1098
  * Re-recorded Broker Pod voices. by @Dzierzan in https://github.com/OpenHV/OpenHV/pull/1099
 
* Higher quality sounds. by @Dzierzan in https://github.com/OpenHV/OpenHV/pull/1046 https://github.com/OpenHV/OpenHV/pull/1050
* Added repairing sounds for Repair Tank and Technician. by @Dzierzan in https://github.com/OpenHV/OpenHV/pull/1063

## Balancing
* Technicians disabled by ECM can't capture buildings. by @GraionDilach in https://github.com/OpenHV/OpenHV/pull/952
* Swapped AA guns for Light and Patrol boats. by @Dzierzan in https://github.com/OpenHV/OpenHV/pull/989
* Allowed Speeder and Banshee to target ground actors. by @Dzierzan in https://github.com/OpenHV/OpenHV/pull/991
* Removed Air from ValidTargets for Speeder and Banshee. by @Dzierzan in https://github.com/OpenHV/OpenHV/pull/994
* Swapped lightning boat and railgun boat . by @Dzierzan in https://github.com/OpenHV/OpenHV/pull/1017
* Increase Banshee and Speeder damages vs non-buildings by @WilliamWeberBerrutti in https://github.com/OpenHV/OpenHV/pull/1040
* Rebranded Medium Boat as Mercenery Boat by @Dzierzan in https://github.com/OpenHV/OpenHV/pull/1061
* Differentiation between the two factions' scouts by @OcelotWalrus in https://github.com/OpenHV/OpenHV/pull/1097 https://github.com/OpenHV/OpenHV/pull/1101
* Televator can now teleport units across the whole map. by @Mailaender 
* Reworked Hacker Tank temporary capturing. by @Mailaender https://github.com/OpenHV/OpenHV/issues/1023
* Hacker tanks are now immune to temporary owner change and ECM tanks immune to ECM. by @Mailaender
* Swapped secondary abilities of ECM and Hacker Tank. @Mailaender

## Fixes
* Don't animate ore purifier & Ore smelter when disabled. by @OcelotWalrus in https://github.com/OpenHV/OpenHV/pull/976
* Fixed diagonal facings on submarine artwork. by @Dzierzan in https://github.com/OpenHV/OpenHV/pull/983
* Gave back heavy machine gun sounds to bunkers. by @Dzierzan in https://github.com/OpenHV/OpenHV/pull/986
* Decreased interval for idle animations of Ore Smelter/Purifier. by @Dzierzan in https://github.com/OpenHV/OpenHV/pull/987
* Updated armament offsets for machine gun pod. by @Dzierzan in https://github.com/OpenHV/OpenHV/pull/988
* Added missing bridge shadows for Harbour map. by @Dzierzan in https://github.com/OpenHV/OpenHV/pull/996
* Fixed aircraft shadows not rotating correctly. by @GraionDilach in https://github.com/OpenHV/OpenHV/pull/1005
* Fixed tiling errors on map Clovergrief. by @SiegeSpud in https://github.com/OpenHV/OpenHV/pull/1003
* Fixed Missile Submarine's generic name. by @OcelotWalrus in https://github.com/OpenHV/OpenHV/pull/1027
* Fixed animals capturing neutral buildings. by @OcelotWalrus in https://github.com/OpenHV/OpenHV/pull/1049
* Overhaul to Ice River mission by @OcelotWalrus in https://github.com/OpenHV/OpenHV/pull/1055
* Allow Synapol AI to produce builders by @OcelotWalrus in https://github.com/OpenHV/OpenHV/pull/1065
* Try to improve performance be decreasing targeting scan intervals. by @GraionDilach in https://github.com/OpenHV/OpenHV/pull/1087
* Fixed Electric Pod outline for north facing. by @Dzierzan in https://github.com/OpenHV/OpenHV/pull/1094
* Fixed Ore smelter/purifier glitching animations when damaged. by @OcelotWalrus in https://github.com/OpenHV/OpenHV/pull/1105
* Fixed bridge shadows being walkable. https://github.com/OpenHV/OpenHV/issues/953 by @Mailaender 
* Fixed Drop Pods being allowed on invalid terrain. https://github.com/OpenHV/OpenHV/issues/944 by @Mailaender 
* Fixed units staying behind resource towers. https://github.com/OpenHV/OpenHV/issues/942 by @Mailaender 
* Fixed units bought at trader not be selectable with counterparts. by @Mailaender 
* Fixed scouts not lifting on their own on map start. https://github.com/OpenHV/OpenHV/issues/1058 by @Mailaender 
* Fixed a crash when planes attacked invalid targets. https://github.com/OpenHV/OpenHV/issues/1060 by @Mailaender 
* Fixed gold balls spawning in resource zones, blocking miners. https://github.com/OpenHV/OpenHV/issues/1075 by @Mailaender 
* Fixed a crash when Drop Pods were deploying into situations where they can't find a path. https://github.com/OpenHV/OpenHV/issues/1067 by @RoosterDragon 
* Fixed skirmish button spamming causing UI glitches. https://github.com/OpenHV/OpenHV/issues/981 by @Mailaender 
* Fixed a crash when aircraft were entering carriers. https://github.com/OpenHV/OpenHV/issues/1006 by @darkademic
* Fixed different build menu hotkeys for PvP and PvE. https://github.com/OpenHV/OpenHV/issues/1037 by @Mailaender 
* Fixed a crash when bots tried to attack with dead units. https://github.com/OpenHV/OpenHV/issues/1076 by @Mailaender 
* Fixed a multiplayer crash around APC ejecting. https://github.com/OpenHV/OpenHV/issues/1089 by @Mailaender 

Based on a [development snapshot](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/300a0c581716c1a122d9726e8759634934aa5e46) of @OpenRA

**Comment by OcelotWalrus** [comment]
You forgot that new Fractional Freezing map by @SiegeSpud.

**Reply by OcelotWalrus** [reply]
You also forgot Bay Under The Rain by myself.

**Reply by Mailaender** [reply]
Added them.
